### PR TITLE
cht-sh: unstable-2019-08-06 -> unstable-2020-08-06

### PR DESCRIPTION
**Pull request by evanjs**
###### Motivation for this change
https://github.com/chubin/cheat.sh/compare/f507ba51d6bc1ae6c7df808cadbe4f8603951b6b...9f99bec1f0293e84d6d8a990a1940c1422e3b0ce

### CODE REVIEW

File: pkgs/tools/misc/cht.sh/default.nix [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
 
   nativeBuildInputs = [ makeWrapper ];
 
   src = fetchFromGitHub {
     owner = "chubin";
     repo = "cheat.sh";
-    rev = "f507ba51d6bc1ae6c7df808cadbe4f8603951b6b";
-    sha256 = "0r7x9a3qzzkbd1m5zdlkpmhx0p7b7ja42190s7fidls3dsm710g0";
+    rev = "9f99bec1f0293e84d6d8a990a1940c1422e3b0ce";
+    sha256 = "sha256:1n4lgzsgg4502zh113d7pb1hw6bykqx6vpfp8j08z7y5clmdiwa6";
```
Comment (fgaz) [inline pkgs/tools/misc/cht.sh/default.nix:20]: Is this a new way to specify hashes or a copypaste error?

Comment (evanjs) [reply pkgs/tools/misc/cht.sh/default.nix]: Typo indeed.
Ironically, they both build fine, which is probably part of the reason I forgot to change it :stuck_out_tongue: 
Thanks!

**Review by fgaz — CHANGES_REQUESTED** [review]
Please use the package attribute in the commit message (`cht-sh`), not the pname

**Comment by evanjs** [comment]
> Please use the package attribute in the commit message (`cht-sh`), not the pname

Will do this going forward. Thanks!

**Comment by fgaz** [comment]
Actually CONTRIBUTING.md is ambiguous about it, but ofborg will only build the package if the commit mentions the attr

**Review by fgaz — APPROVED** [review]
LGTM

**Pull request merged**